### DISCUSSION TITLE
Lazy loading into existing rows

### DISCUSSION THREAD

**Discussion by dev-guy** [TanStack/table#2965] [Q&A]
I am using react table with react query; both are excellent. The rows in the table contain columns that are derived by invoking REST endpoints in the background. In other words, some cells contain the value “pending” and are updated within a few seconds. Is there an example that shows how to do this?

**Comment by WeiTangLau** [comment]
I think that you are looking for this 
https://react-table.tanstack.com/docs/examples/sub-components-lazy
@dev-guy
